PSI Food waste/Pages/Forms/RestaurantVerified.cshtml

```diff
@@ -48,9 +48,16 @@
             <td>@product.Size</td>
             <td>@Model.GlutenFreeText(product)</td>
             <td>
-                <form method="post" asp-page-handler="Delete" asp-route-id="@product.PrID">
-                    <button class="btn btn-danger">Delete</button>
-                </form>
+                <div class="btn-group">
+                    <div class="btn-sm">
+                        <a asp-page="./EditProduct" asp-route-id="@product.PrID" class="btn btn-primary">Edit</a>
+                    </div>
+                    <div class="btn-sm">
+                        <form class="align-content-end" method="post" asp-page-handler="Delete" asp-route-id="@product.PrID">
```

Comment:
This wont work because the rout name has to match the variable name so change it to asp-route-prid